### PR TITLE
Implement websocket to perform Operation over resources

### PR DESCRIPTION
**Pull request by openwarrior**
- Add properties and params to Operation model.
- Add utility classes to perform websocket connection related tasks.
-  SSL compatiblity is not working yet.
- Operations and custom params on operations are completely working.

### CODE REVIEW

File: mobile/src/main/java/org/hawkular/client/android/fragment/ConfirmOperationFragment.java [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+                    case "float": {
+                        row = (TableRow) LayoutInflater.from(getActivity()).inflate(R.layout.row_edit, null);
+                        EditText data = (EditText) row.findViewById(R.id.data);
+                        data.setText(operationParameter.getDefaultValue());
+                        data.setInputType(InputType.TYPE_CLASS_NUMBER | InputType.TYPE_NUMBER_FLAG_DECIMAL);
+                        break;
+                    }
+
+                    default: {
```
Comment (pilhuhn) [inline mobile/src/main/java/org/hawkular/client/android/fragment/ConfirmOperationFragment.java]: I guess this is meant for boolean. Better have this explicit as `case "boolean":` and use the default for cases where the incoming data is not known. As is it works, but it is not obvious when you read the code.

**Comment by openwarrior** [comment]
on it

**Comment by pilhuhn** [comment]
Thanks @anuj1708 that is great.

**Comment by openwarrior** [comment]
@pilhuhn Thanks

**Pull request merged**